.ci/cico_build.sh

```diff
@@ -30,8 +30,15 @@ if [[ "$DeveloperBuild" != "true" && "$PR_CHECK_BUILD" != "true" && "$USE_CHE_LA
 
   set -x
   yum -y update
-  yum -y install centos-release-scl scl-utils java-1.8.0-openjdk-devel git patch bzip2 golang
-  yum -y install rh-maven33 rh-nodejs8
```

Comment:
I believe we need to consider installing rh-maven36  (centos community build)

Reply:
It was a pain in the ass since it connected to java8. I was not able to manage that.